imops/morphology.py

```diff
@@ -1,8 +1,17 @@
-from typing import Callable
+from typing import Callable, Tuple, Union
 from warnings import warn
 
 import numpy as np
-from scipy.ndimage import generate_binary_structure
+from edt import edt
+from scipy.ndimage import distance_transform_edt as scipy_distance_transform_edt, generate_binary_structure
+from scipy.ndimage._nd_image import euclidean_feature_transform
```

Comment:
unsafe import. need a fallback at least

Reply:
For python 3.6-3.12 it works ok. And I don't know what fallback to use

Reply:
ok. let's wait until this changes. then we'll patch it